docs/getting-started/security-ui.asciidoc

```diff
@@ -37,15 +38,15 @@ The {security-app} contains the following pages that enable analysts to view, an
 * Host isolation exceptions
 * Blocklist
 
-Pages are grouped into four main sections within the navigation pane -- Detect, Explore, Investigate, and Manage. Each section supports a different part of your workflow and describes actions you can perform in the {security-app}.
+Pages are grouped into four main sections within the navigation pane -- *Detect*, *Explore*, *Investigate*, and *Manage*. Each section supports a different part of your workflow and provides actions you can perform in the {security-app}.
 
-From the *Detect* section, you can access pages that allow you to view, create, and manage alerts, rules, and rule exceptions.
+From the *Detect* section, access pages that allow you to view, create, and manage alerts, rules, and rule exceptions.
 
-Under *Explore*, you access pages that provide key metrics about your hosts and networks.
+Under *Explore*, access pages that provide key metrics about your hosts and networks.
 
-The *Investigate* section gives you access to timelines and cases.
+The *Investigate* section gives you access to Timelines and cases.
 
-Pages under the *Manage* section allow to view and manage hosts that are running Endpoint Security. You can also view and manage trusted applications and event filters from pages in this section.
+Pages under the *Manage* section allow you to view and manage hosts that are running Endpoint Security. You can also view and manage trusted applications and event filters from pages in this section.
```

Comment:
Even if we keep these as full sentences, I'd still omit the last sentence here -- it's outdated and I don't think we need to list all the Endpoint artifacts here, just mention that *Manage* is related to Endpoint Security.
```suggestion
Pages under the *Manage* section allow you to view and manage hosts that are running {endpoint-sec}.
```